misc/python/materialize/mzcompose/__init__.py

```diff
@@ -79,6 +79,7 @@ def get_default_system_parameters(
         "cluster_always_use_disk": "true",
         "compute_dataflow_max_inflight_bytes": "134217728",  # 128 MiB
         "compute_hydration_concurrency": "2",
+        "compute_replica_expiration_offset": "3d",
```

Comment:
Maybe more than three days, zippy already takes close to two days. But no strong feeling either way.

Reply:
@def- should we increase the offset here?

Reply:
I'd prefer to keep it at 3d and limit Zippy if it exceeds 2 day runtime.